DeAceroDLabs/marketplace-frontend
# Feature/dmp 434

## What is done

[[DMP-434] Page not found](https://deacero.atlassian.net/browse/DMP-434)

## How is done

- Created a `PageNotFound` component that will be used when the user is logged and the page is not found with a button that can redirect to the home screen.
- This button needed another style it´s larger than primary.

## How to test

1. Run the project via `npm start`
2. You should see the Home Screen. 
3. Go to the end of the page and click on the button ayuda, there you will see the `PageNotFound` component.
